proto/lookout/sdk/service_data.proto

```diff
@@ -41,40 +42,69 @@ message File {
     string hash = 3;
     // Raw content of the file.
     bytes content = 4;
-    // UAST.
+    // UAST is a Babelfish v1 UAST.
     gopkg.in.bblfsh.sdk.v1.uast.Node uast = 5 [(gogoproto.customname) = "UAST"];
     // Programming/data/markup language of the file.
     string language = 6;
 }
 
+// Change contains two version of a File in a revision range.
 message Change {
     option (gogoproto.sizer) = false;
     option (gogoproto.protosizer) = true;
     
+    // Base is the file version at the base of the revision range.
     File base = 1;
+    // Head is the file version at the head of the revision range.
     File head = 2;
 }
 
+// ChangesRequest defines a request of Changes to the Data service.
 message ChangesRequest {
+    // Base of the revision range.
     ReferencePointer base = 1;
+    // Head of the revision range.
     ReferencePointer head = 2;
+    // IncludePattern is a regexp. Only the files with matching filenames will
+    // be included.
     string include_pattern = 3;
+    // ExcludePattern is a regexp. Any files with matching filenames will be
+    // excluded.
     string exclude_pattern = 4;
+    // ExcludeVendored will exclude any files inside a vendor path.
```

Comment:
This comment isn't completely correct. It excludes not only files inside a vendor path but vendor files itself too like `whatever.min.js` or `Vagrantfile` or `normalize.(css|less|scss|styl)` and many others.
It might be a good idea to add reference to enry/linguist here.